static/openapi/prerendering-spec.json

```diff
@@ -1569,34 +954,35 @@
             "in": "path",
             "name": "type",
             "required": true,
-            "description": "Interaction type (scroll, click, hover, etc...)"
+            "description": "Interaction type."
```

Comment:
Would an enum define what the types are?

Reply:
Enum might be defined already, can't see it in the diff.  Ignore then.

Reply:
Yeah, the enum is defined. I also removed the examples as all options appear in the documentation page.